rust-toolchain.toml

```diff
@@ -0,0 +1,4 @@
+[toolchain]
+channel = "stable"
+targets = [ "wasm32-unknown-unknown" ]
+components = [ "rust-src", "rust-analyzer" ]
```

Comment:
So after I first did:

```
cargo run -p carlo -- --help
info: syncing channel updates for 'stable-x86_64-unknown-linux-gnu'
info: latest update on 2023-06-01, rust version 1.70.0 (90c541806 2023-05-31)
info: downloading component 'rust-analyzer'
info: downloading component 'rust-src'
info: downloading component 'rust-std' for 'wasm32-unknown-unknown'
info: downloading component 'rust-std' for 'aarch64-linux-android'
info: downloading component 'rust-std' for 'armv7-linux-androideabi'
info: downloading component 'rust-std' for 'riscv32imc-unknown-none-elf'
```

I get all this downloading stuff for some reason. Do we need the `channel` argument there? It's pretty annoying as it made my rust analyzer go bonkers recompiling everything again. I guess I was using nightly. We need to pin to specific rustc versions for actual carol guest development though but I was hoping carlo would do that. Maybe for developing carol we can leave it without an explicit channel.